src/week.jsx

```diff
@@ -51,8 +58,17 @@ export default class Week extends React.Component {
   renderDays = () => {
     const startOfWeek = this.props.day.clone().startOf('week')
     const days = []
+    const weekNumber = this.formatWeekNumber(startOfWeek)
     if (this.props.showWeekNumber) {
-      days.push(<WeekNumber key="W" weekNumber={this.formatWeekNumber(startOfWeek)} />)
+      days.push(
+        <WeekNumber
+            key="W"
+            weekNumber={weekNumber}
+            onClick={this.props.onWeekSelect
+              ? this.handleWeekClick.bind(this, startOfWeek, weekNumber)
+              : undefined
+            } />
```

Comment:
The undefined is to determine whether there is a callback or not. The WeekNumber component uses this prop to assign a CSS class that makes the UI responsive to mouse over action. Simply passing a local method would yield always true.